.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+# Motivation
+
+[Include the reason behind these changes and any relevant context.]
+
+# Description
+
+[Provide a detailled explanation of the modifications you have made. Link any related issues.]
+
+# Testing
+
+[When applicable, detail the testing you have performed to ensure that these changes function as intended. Include information about any added tests.]
+
+# Impact
+
+[Discuss the impact of your modifications on ArmoniK. This might include effects on performance, configuration, documentation, new dependencies, or changes in behaviour.]
```

Comment:
```suggestion
<!--
Discuss the impact of your modifications on ArmoniK. This might include effects on performance, configuration, documentation, new dependencies, or changes in behaviour.]
-->
```